setup.py

```diff
@@ -39,7 +39,7 @@ def load_requirements(requirements_folder):
     edx_platform_version = get_edx_platform_version()
 
     if edx_platform_version == OPEN_EDX_GINKGO_VERSION:
-        requirements_path = '{}/{}'.format(requirements_folder, 'ginkgo.in')
+        requirements_path = '{}/{}'.format(requirements_folder, 'ginkgo.txt')
```

Comment:
Why did you apply this change? @cocococosti

Reply:
this was in the original commit https://github.com/proversity-org/openedx-proversity-reports/commit/d27207733f16481e8a573d018c0d74d70be2db80 @Squirrel18

Reply:
Ok, but you are adding it for Ironwood, and that change was for Ginkgo, so... @cocococosti